lib/apiservers/engine/backends/archive_test.go

```diff
@@ -235,7 +235,7 @@ func TestFindArchiveReaders(t *testing.T) {
 					rebase:  "etc",
 					strip:   "etc",
 					exclude: []string{""},
-					include: "etc/",
+					include: "etc",
```

Comment:
can we have a test that explicitly has the trailing `/` to assert behaviour in that case.